jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererUtil.kt

```diff
@@ -94,18 +101,41 @@ object CodeWhispererUtil {
                 }
             } catch (e: SsoOidcException) {
                 runInEdt {
-                    SsoConnectionExpiredDialog(project, connection).show()
+                    notifyConnectionExpired(project, connection)
                     callback()
                 }
             }
         } else if (state == BearerTokenAuthState.NOT_AUTHENTICATED) {
             runInEdt {
-                SsoConnectionExpiredDialog(project, connection).show()
+                notifyConnectionExpired(project, connection)
                 callback()
             }
         }
     }
 
+    private fun notifyConnectionExpired(project: Project, connection: BearerSsoConnection?) {
+        connection ?: return
+        logoutFromSsoConnection(project, connection) {
+            UiTelemetry.click(project, "signout_codewhisperer_expired_connection")
```

Comment:
This feels out of place. Has the user clicked anything when this code is executed?

Reply:
in fact, not at all, we logout on behalf of them basically, but we don't have other types of telemetry for now, or I can just remove this part?

Reply:
as the name suggested, i think i should remove the telemetry report here then add others later